src/Neo/Cryptography/Base58.cs

```diff
@@ -27,6 +28,9 @@ public static class Base58
         /// </summary>
         public const string Alphabet = "123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz";
 
+        private static readonly char s_zeroChar = Alphabet[0];
+        private static readonly Lazy<IReadOnlyDictionary<char, int>> s_alphabetDic = new(() => Enumerable.Range(0, Alphabet.Length).ToDictionary(t => Alphabet[t], t => t));
```

Comment:
This cost more in the current code in resources of memory management for the `GC` in `dotnet`.